cse/CommentReader.py

```diff
@@ -1,51 +1,75 @@
 import csv
 import os
-
+import warnings
 
 class CommentReader(object):
 
     __delimiter = ''
-    __filepath = ""
-    __file = None
-    __reader = None
-
-
-    def __init__(self, filepath, delimiter=','):
+    __commentsFilepath = ""
+    __commentsFile = None
+    __commentReader = None
+    
+    __articlesFilepath = None
+    __articlesFile = None
+    __articlesReader = None
+    __currentArticle = None
+
+    __authorsFilepath = None
+    __authorsFile = None
+    __authorsReader = None
+    __authors = None
+
+
+    def __init__(self, commentsFilepath, arcticlesFilepath, authorsFilepath, delimiter=','):
         self.__delimiter = delimiter
-        self.__filepath = filepath
-
+        self.__commentsFilepath = commentsFilepath
+        self.__articlesFilepath = arcticlesFilepath
+        self.__authorsFilepath = authorsFilepath
 
     def open(self):
-        if not os.path.exists(os.path.dirname(self.__filepath)):
+        if not os.path.exists(os.path.dirname(self.__commentsFilepath)):
             raise Exception("file not found!")
 
-        self.__file = open(self.__filepath, 'r', newline='')
-        self.__reader = csv.reader(self.__file, delimiter=self.__delimiter)
+        self.__commentsFile = open(self.__commentsFilepath, 'r', newline='')
+        self.__commentReader = csv.reader(self.__commentsFile, delimiter=self.__delimiter)
+
+        self.__articlesFile = open(self.__articlesFilepath, 'r', newline='')
+        self.__articlesReader = csv.reader(self.__articlesFile, delimiter=self.__delimiter)
+
+        self.__authorsFile = open(self.__authorsFilepath, 'r', newline='')
+        self.__authorsReader = csv.reader(self.__authorsFile, delimiter=self.__delimiter)
+
         return self
 
 
     def close(self):
-        self.__file.close()
+        self.__commentsFile.close()
+        self.__articlesFile.close()
+        self.__authorsFile.close()
 
 
     def readAllData(self):
+        raise DeprecationWarning("deprecated and not longer supported")
+        if not self.__authors:
+            self.__loadAuthors()
+        return
```

Comment:
I don't get it. If you want to leave the method because it is no longer supported, why checking the authors-field?
If you want to leave the method if there are no authors, then line 55 has the wrong indentation.

Reply:
there is an exception before that. I forgot to remove the method body

Reply:
please do that